content/docs/reconciliation.md

```diff
@@ -59,7 +59,7 @@ When comparing two React DOM elements of the same type, React looks at the attri
 <div className="after" title="stuff" />
 ```
 
-By comparing these two elements, React knows to only modify the `className` on the underlying DOM node.
+By comparing these two elements, React knows to only modify the `class` attribute on the underlying DOM node.
```

Comment:
While React does set the attribute, that's only determined by the renderer not the reconciler. `react-dom` could very well decide to set the property instead, or even use `classList`. So I don't think this distinction is relevant to explaining how reconciliation works.